test/integration/remote-config.spec.ts

```diff
@@ -57,16 +57,22 @@ const VALID_PARAMETER_GROUPS = {
   },
 };
 
-const VALID_CONDITIONS: admin.remoteConfig.RemoteConfigCondition[] = [{
-  name: 'ios',
-  expression: 'device.os == \'ios\'',
-  tagColor: 'INDIGO',
-},
-{
-  name: 'android',
-  expression: 'device.os == \'android\'',
-  tagColor: 'GREEN',
-}];
+const VALID_CONDITIONS: admin.remoteConfig.RemoteConfigCondition[] = [
+  {
+    name: 'ios',
+    expression: 'device.os == \'ios\'',
+    tagColor: 'INDIGO',
+  },
+  {
+    name: 'android',
+    expression: 'device.os == \'android\'',
+    tagColor: 'GREEN',
+  },
+];
+
+const VALID_VERSION: admin.remoteConfig.Version = {
```

Comment:
Just wondering. Are these explicit type annotations really required?

Reply:
I removed the type annotations for `VALID_VERSION`. It looks like we need the type annotation for `VALID_CONDITIONS` unless I use a `const INDIGO: admin.remoteConfig.TagColor = 'INDIGO'` for tag color.